easybuild/easyconfigs/g/Graphviz/Graphviz-2.40.1-foss-2018b.eb

```diff
@@ -0,0 +1,83 @@
+# easybuild easyconfig
+#
+# Graphviz has moved to gitlab and it is no longer possible to get tagged
+# versions.  There is an open ticket this issue.
+
+easyblock = 'ConfigureMake'
+
+name = 'Graphviz'
+version = '2.40.1'
+
+homepage = 'http://www.graphviz.org/'
+description = """Graphviz is open source graph visualization software. Graph
+ visualization is a way of representing structural information as diagrams of
+ abstract graphs and networks. It has important applications in networking,
+ bioinformatics,  software engineering, database and web design, machine
+ learning, and in visual interfaces for other technical domains."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://fossies.org/linux/misc/']
+sources = ['graphviz.tar.gz']
+patches = [
+    'Graphviz-2.38.0_icc_sfio.patch',
+    'Graphviz-2.40.1_icc_vmalloc.patch',
+    'Graphviz-2.40.1_Qt5.patch',
+    'Graphviz-2.40.1_skip-install-data-hook.patch',
+]
+checksums = [
+    'ca5218fade0204d59947126c38439f432853543b0818d9d728c589dfe7f3a421',  # graphviz-2.40.1.tar.g
+    '393a0a772315a89dcc970b5efd4765d22dba83493d7956303673eb89c45b949f',  # Graphviz-2.38.0_icc_sfio.patch
+    '813e6529e79161a18b0f24a969b7de22f8417b2e942239e658b5402884541bc2',  # Graphviz-2.40.1_icc_vmalloc.patch
+    'f88ef7bcdb7cdfa2cda89c4681db3fecfb0e37955d52c0d4ef5bcffe5b41eb55',  # Graphviz-2.40.1_Qt5.patch
+    '8685c67b3c83f814cdf87f626905695b249eb7c3f64a2b1cdb79733b7297a4a4',  # Graphviz-2.40.1_skip-install-data-hook.patch
+]
+
+dependencies = [
+    ('cairo', '1.14.12'),
+    ('expat', '2.2.5'),
+    ('freetype', '2.9.1'),
+    ('fontconfig', '2.13.0'),
+    ('Ghostscript', '9.23'),
+    ('GObject-Introspection', '1.54.1', '-Python-2.7.15'),
+    ('GTS', '0.7.6'),
+    ('Java', '1.8', '', True),
+    ('libpng', '1.6.34'),
+    ('Pango', '1.42.4'),
+    ('Perl', '5.28.0'),
+    ('Qt5', '5.10.1'),
+    ('Tcl', '8.6.8'),
+    ('zlib', '1.2.11'),
+]
+
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('SWIG', '3.0.12', '-Python-2.7.15'),
+    ('pkg-config', '0.29.2'),
+]
+
+preconfigopts = './autogen.sh && '
+
+configopts = '--enable-guile=no --enable-lua=no --enable-ocaml=no '
+configopts += '--enable-r=no --enable-ruby=no '
+configopts += '--enable-ruby=no --enable-php=no '
```

Comment:
@fizwit `--enable-ruby=no` is duplicated

Reply:
ok, removing unnecessary ruby